### PR TITLE
inlined some CSS

### PR DESCRIPTION
**Pull request by fabpot**
| Q | A |
| --- | --- |
| Branch? | master |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR | n/a |

This PR has several goals:
- It fixes the discrepancy between where CSS files were stored (FrameworkBundle) and where there were used (TwigBundle);
- It removes the need to install the assets to get exceptions (think Silex, micro-kernel-based-edition, ...);
- It makes things consistent with how we already manage images for exceptions (they are inlined);
- It makes things consistent with how we manage assets for the web profiler.

### CODE REVIEW

**Comment by fabpot** [comment]
If you are wondering why we did not cleanup this before, that's because those CSSes were shared between the web profiler and the twig exception files, but that's not the case anymore since the redesign of the web profiler.

File: src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception_full.html.twig [hunk 1]
```diff
@@ -1,7 +1,136 @@
 {% extends '@Twig/layout.html.twig' %}
 
 {% block head %}
-    <link href="{{ absolute_url(asset('bundles/framework/css/exception.css')) }}" rel="stylesheet" type="text/css" media="all" />
+    <style>
+        .sf-reset .traces {
+            padding-bottom: 14px;
```
Comment (Koc) [inline src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception_full.html.twig]: is it better use separate css file and include it here?

Comment (fabpot) [inline src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception_full.html.twig]: What would be the benefit? It means adding a controller to handle it.

Comment (Koc) [inline src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception_full.html.twig]: we do same for svg icons inlining without controller (use twig include):
- https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/TwigBundle/Resources/views/layout.html.twig#L15
- https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/results.html.twig#L38

Comment (fabpot) [inline src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception_full.html.twig]: I agree, this is indeed what is done elsewhere, but here, I don't want to "pollute" the namespace as there are only exceptions templates, and creating a new directory up sounds like not really useful.

**Comment by nicolas-grekas** [comment]
:+1:

**Comment by fabpot** [comment]
added the change to the CHANGELOG file

**Pull request merged**